scripts/hooks/reload-exec-daemon-hooks.sh

```
@@ -0,0 +1,46 @@
+# Optional env:
+#   EXEC_DAEMON_PORT   default 26053
+#   EXEC_DAEMON_TOKEN  if unset, scraped from the running exec-daemon cmdline
+
+set -euo pipefail
+
+PORT="${EXEC_DAEMON_PORT:-26053}"
+TOKEN="${EXEC_DAEMON_TOKEN:-}"
+
+if [ -z "${TOKEN}" ]; then
+  # Best-effort: parse --auth-token from the live exec-daemon process.
+  CMDLINE="$(tr '\0' ' ' </proc/$(pgrep -f '/exec-daemon/index.js serve' | head -n1)/cmdline 2>/dev/null || true)"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)